<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(244, 247, 250);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 83 12pt &quot;Segoe UI&quot; bold;</string>
   </property>
   <property name="text">
    <string>Calibration : Pixel to Centimeters</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load_calib_img">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 10pt &quot;Segoe UI&quot; bold;</string>
   </property>
   <property name="text">
    <string>Load Calibration Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 10pt &quot;Segoe UI&quot; bold;</string>
   </property>
   <property name="text">
    <string>Enter Value Manually</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pixel_cm">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 10pt &quot;Segoe UI&quot; bold;</string>
   </property>
   <property name="text">
    <string>Pixels = 1 cm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 10pt &quot;Segoe UI&quot; bold;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
